### PR TITLE
#9: DropdownMenu should be attachable to any component passed as children (closes #9)

### LINKED ISSUE #9: DropdownMenu should be attachable to any component passed as children
Requirements
- `DropdownMenu` (thus `Menu`) should create menu just by `options`
- `children` must be the component which is attached to
- if `children` is just an icon, one can pass an `iconClassName` string as a shortcut (current implementation)

### PR DESCRIPTION
**Pull request by francescogior**
Issue #9

### CODE REVIEW

**Comment by giogonzo** [comment]
Looks good! remember tests & lint, and I'd like to see it working on lol first

**Comment by francescogior** [comment]
lint was okay, test not okay, gonna fix

**Comment by francescogior** [comment]
Fixed tests. To see it working on labonline you can checkout the https://github.com/buildo/labonline/issues/552 branch

**Comment by giogonzo** [comment]
@FrancescoCioria care to review this quickly?

File: src/DropdownMenu/dropdownMenu.scss [hunk 1]
```diff
@@ -2,6 +2,9 @@
 @import '~sass-flex-mixins/src/sass-flex-mixins.scss';
 
 .dropdown-menu {
+  position: relative;
+  @include display_flex();
+  @include flex_align-items(center);
```
Comment (FrancescoCioria) [inline src/DropdownMenu/dropdownMenu.scss]: why aren't we using `FlexView`?

File: src/DropdownMenu/DropdownMenu.js [hunk 2]
```diff
@@ -31,13 +33,20 @@ export default class DropdownMenu extends React.Component {
...
+    leaveOpenOnClick: false
   }
 
   toggleMenu = () => {
     this.setState({ isOpen: !this.state.isOpen });
   }
 
+  onMenuClick = () => {
+    if (!this.props.leaveOpenOnClick) {
+      this.toggleMenu();
```
Comment (FrancescoCioria) [inline src/DropdownMenu/DropdownMenu.js]: I'm confused...
if `initiallyOpen=false` and `leaveOpenOnClick=true` how do you open it? I mean, `this.toggleMenu` is never called, right?

Comment (giogonzo) [inline src/DropdownMenu/DropdownMenu.js]: and also, how do you close it after it is opened? There's no way of forcing it to close from user perspective?

It think we can leave this option (`leaveOpenOnClick`) out for now, and add it with proper api if/when it is needed (not sure what the use case was @francescogior )

I'll take care of this

File: src/DropdownMenu/dropdownMenu.scss (hunk 1 above)
Comment (giogonzo) [inline src/DropdownMenu/dropdownMenu.scss]: :+1: , I'll make the switch

File: src/DropdownMenu/DropdownMenu.js (hunk 2 above)
Comment (francescogior) [inline src/DropdownMenu/DropdownMenu.js]: The idea was that sometimes you don't want the menu to hide after the click on an option/action.
The menu could always be toggled with the toggleButton (the ellipsis icon) and, at least by default, by any click outside of it.

**Pull request merged**